lib/deployer.js

```diff
@@ -100,7 +100,7 @@ var Actions = {
       return new Promise(function(accept, reject) {
         Require.exec({
           file: file,
-          contracts: Object.keys(deployer.known_contracts).map(function(key) {
+          contracts_build_directory: Object.keys(deployer.known_contracts).map(function(key) {
```

Comment:
I think this should be the other way around. According to the require.exec() function in require.js, it accepts an optional array of contracts. If that array is specified, it is used instead of the contracts under options.contracts_build_directory (implicitly inside contracts.provision).

So the fix would be to add "contracts" to the expected options array used in require.exec().